psnineplus.js

```diff
@@ -457,7 +457,7 @@
             }
             games[gameId].map(element => {
                 if (element.trophy === $(el).attr('href')) {
-                    $(el).next().find('a').append(`<div class="fa-check-circle"></div>&nbsp;<em class="alert-success pd5" style="border-radius: 3px;">${element.earned}</em>`);
+                    $(el).next().find('a').slice(0, 1).append(`<div class="fa-check-circle"></div>&nbsp;<em class="alert-success pd5" style="border-radius: 3px;">${element.earned}</em>`);
```

Comment:
```suggestion
                    $(el).next().find('a:first-of-type').append(`<div class="fa-check-circle"></div>&nbsp;<em class="alert-success pd5" style="border-radius: 3px;">${element.earned}</em>`);
```

这样的话少一步操作，可以帮忙测试一下是否OK👍没问题的话就Merge了，顺便版本号记得加一下

Reply:
interesting
本地测试first-of-type还是会返回全部

Reply:
那就这样行了，
https://github.com/swsoyee/psnine-enhanced-version/blob/master/psnineplus.js#L4 （GreasyFork更新用）
https://github.com/swsoyee/psnine-enhanced-version/blob/master/package.json#L3 （虽然现在没用上）
这两个提一个版本就好